lua/astrocommunity/media/vim-wakatime/init.lua

```diff
@@ -1 +1,7 @@
-return { "wakatime/vim-wakatime", event = "User AstroFile" }
+return { 
+  "wakatime/vim-wakatime",
+  event = "User AstroFile",
```

Comment:
it is weird how in go.nvim we adhere to the readme of go.nvim plugin's setup instructions but here we don't. how is this determined? this stems from a previous issue I created

Reply:
https://github.com/AstroNvim/astrocommunity/issues/560